examples/smtpServerAvailability.js

```diff
@@ -0,0 +1,35 @@
+/* 
+    A script that checks availability of an SMTP Server by connecting to the service and evaluating the response code.
+*/
+
+import { By, Key, until } from 'selenium-webdriver';
+import { net, driver, markers, credentials, downloads, transaction, authentication } from 'thousandeyes';
+import assert from 'assert';
+runScript();
+
+async function runScript() {
+    // Set host and port
+    let host = 'smtp.gmail.com';
+    let port = 587;
+    let resp = await callServer(host, port);
```

Comment:
```suggestion
    await callServer(host, port);
```
This function does not return anything, and the variable resp is not used anyway. No need to assign to a variable